std_msgs/Header last_header
std_msgs/Header first_header
ScannedObj[] scannedObjList
